### DISCUSSION TITLE
How to set the window in the middle of the desktop?

### DISCUSSION THREAD

**Discussion by push-and-pop** [hecrj/iced#1058] [Q&A] [answered]
```
window: window::Settings {
            size: (800, 600),
            ..Default::default()
        },
```
I did not find it

**Comment by hecrj** [comment, marked as answer]
If you rely on `master`, you should be able to use the `position` field in `window::Settings`:

https://github.com/iced-rs/iced/blob/35c4ad6dd950fa91d181f4d3253fcb486bcf1b11/src/window/settings.rs#L9-L10

And set it to `Position::Centered`:

https://github.com/iced-rs/iced/blob/35c4ad6dd950fa91d181f4d3253fcb486bcf1b11/src/window/position.rs#L6-L7

**Comment by push-and-pop** [comment]
![image](https://user-images.githubusercontent.com/80676432/134861178-c5b7e491-91b2-4d80-a36c-d96a258e17d8.png)

![image](https://user-images.githubusercontent.com/80676432/134861366-082f9156-85b7-4336-abfd-8e10d63808dd.png)

Hello, I don’t seem to find this position

**Reply by hecrj** [reply]
This is not available in the latest crates.io release, but only `master` for now.

**Reply by push-and-pop** [reply]
Ok, thank you still
